packages/app-bridge-theme/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @frontify/app-bridge-theme
 
+## 0.0.0-alpha.1
+
+### Patch Changes
+
+- [#960](https://github.com/Frontify/brand-sdk/pull/960) [`7a88fb5`](https://github.com/Frontify/brand-sdk/commit/7a88fb512a8209ab377ef12a70e2c8484d5b6799) Thanks [@anxobotana](https://github.com/anxobotana)! - refactor(AppBridgeTheme): version number change
+
 ## 1.0.0-alpha.2
```

Comment:
Should we include the changes fro this version in `0.0.0-alpha.1` and remove version `1.0.0-alpha.2`? Kinda weird that 1.0.0-alpha.2 has `initial alpha release` and 0.0.0-alpha.1 has `version number change`

Reply:
It is because I just cancelled the release job, didn't do any cleanup, we can add a commit here which cleanups up that

Reply:
on it